samtranslator/swagger/swagger.py

```diff
@@ -612,14 +612,16 @@ def set_path_default_authorizer(  # noqa: too-many-branches
                 if "AWS_IAM" in method_definition["security"][0]:
                     self.add_awsiam_security_definition()
 
-    def set_path_default_apikey_required(self, path: str) -> None:
+    def set_path_default_apikey_required(self, path: str, AddApiKeyRequiredToCorsPreflight: bool = True) -> None:
```

Comment:
Prefer consistent naming for regular variables, can be confusing to see at a glance otherwise.

Reply:
What's a better name I could use?

Reply:
Use `snake_case`. Here the name is quite complex (I assume only because of precedent with similar name). Maybe something like `require_options_api_key`?

Reply:
Should I do this in the api_generator.py file too?